stripe/src/main/java/com/stripe/android/checkout/CheckoutPaymentMethodsListFragment.kt

```diff
@@ -20,26 +19,10 @@ internal class CheckoutPaymentMethodsListFragment : Fragment(R.layout.fragment_c
         val binding = FragmentCheckoutPaymentMethodsListBinding.bind(view)
         binding.recycler.layoutManager = LinearLayoutManager(requireContext(), LinearLayoutManager.HORIZONTAL, false)
 
-        val args: CheckoutActivityStarter.Args? = CheckoutActivityStarter.Args.fromIntent(requireActivity().intent)
-        if (args == null) {
-            viewModel.onError(IllegalStateException("Missing activity args"))
-            return
-        }
-
-        viewModel.getPaymentMethods(
-            args.customerId,
-            args.ephemeralKey
-        ).observe(viewLifecycleOwner) { result ->
-            result.fold(
-                onSuccess = {
-                    binding.recycler.adapter = CheckoutPaymentMethodsAdapter(it) {
-                        viewModel.transitionTo(CheckoutViewModel.TransitionTarget.AddCard)
-                    }
-                },
-                onFailure = {
-                    viewModel.onError(it)
-                }
-            )
+        viewModel.getPaymentMethods(requireActivity()).observe(viewLifecycleOwner) {
```

Comment:
I recommend using the nullable version and returning an error to avoid our SDK causing a crash.

Reply:
the nullable version of `activity`?